tests/schemas/valid/test_response_metadata_value_source.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
Not a blocker but this could be added to one of the existing schemas like `test_string_transforms.json`.

Reply:
Moved response metadata transform to `test_string_transforms.json` and deleted the new schema, thanks!